### ISSUE TITLE
[Angular] Text and Image Components are not visible in authoring mode

### ISSUE THREAD

**Issue by tinomthomas** [Adobe-Marketing-Cloud/aem-guides-wknd-events#83]
**Local Environment**
Before filing an issue verify that the prerequisite tools/software are installed:

* AEM 6.4 + SP2
* Java 1.8
* Apache Maven (3.3.9 or newer)
* Node.js v10+
* npm 6+

**Describe the issue**
Text and Image Components are not visible in authoring mode. And I am not able to see the components in the side panel as well. The interesting thing is that there are no errors in Chrome's dev tools while loading the URL http://localhost:4502/editor.html/content/wknd-events/angular/home.html

**Where in the tutorial are you experiencing an issue?**

In Chapter 3 while reviewing the Text and Image components in AEM

**Screenshots**
![Capture1](https://user-images.githubusercontent.com/38363530/80829724-586a6100-8bf0-11ea-8ef2-12f022eb6bd7.PNG)
![Capture2](https://user-images.githubusercontent.com/38363530/80829760-691ad700-8bf0-11ea-8340-c429c2ece7ff.PNG)

**Comment by godanny86** [comment]
hi @tinomthomas is the .model.json returning as expected? Its possible you are hitting: https://github.com/Adobe-Marketing-Cloud/aem-guides-wknd-events/issues/82 if you have Core Components v 2.8.0 installed...

**Comment by tinomthomas** [comment]
Hi @godanny86 , Yes the angular.model.json is returning as expected. And the installed Core component version is 2.3.2. Upon further investigating using Chrome dev tools, I can see that the angular components are available in the DOM but they are not visible at the screen. Attaching the screenshots
![image](https://user-images.githubusercontent.com/38363530/80892958-dd2faa80-8cd6-11ea-818b-4831c87292e6.png)
![image](https://user-images.githubusercontent.com/38363530/80892982-0d774900-8cd7-11ea-9db0-5c01281ef6f2.png)

**Comment by godanny86** [comment]
hmm interesting. What happens if you deploy the chapter 3 solution? https://github.com/Adobe-Marketing-Cloud/aem-guides-wknd-events/tree/angular/chapter-3

**Comment by tinomthomas** [comment]
Hi @godanny86, Sorry for the late reply. The result is the same

**Comment by tinomthomas** [comment]
Hi @godanny86  I think I have hit the [bug 59](https://github.com/Adobe-Marketing-Cloud/aem-guides-wknd-events/issues/59#issuecomment-520358232). Is there anything I can do to make it work?

**Comment by godanny86** [comment]
Can you remind me what version of AEM you are on? Are you able to update to the latest service pack with AEM 6.4 or 6.5?

**Comment by tinomthomas** [comment]
Yes. I am using 6.4. Updating to the latest is a bit complicated here. But I will try that also
